packages/backend/server/routes/stats.get.ts

```diff
@@ -0,0 +1,74 @@
+export default eventHandler(async (event) => {
+  try {
+    const binding = useBinding(event);
+
+    let cursor: string | undefined = undefined;
+    let totalObjects = 0;
+    let totalSize = 0; 
+    const orgs = new Set<string>();
+    const packages = new Set<string>();
+    const commits = new Set<string>(); 
+
+    const prefix = `${usePackagesBucket.base}:`;
+
+    do {
+      const result = await binding.list({ prefix, cursor });
+
+      const keys = result.objects.map((obj) => obj.key);
+      totalObjects += keys.length;
+
+      for (const obj of result.objects) {
+        const key = obj.key;
+        const size = obj.size || 0;
+        totalSize += size;
+
+        if (!key.startsWith(prefix)) {

```

Comment:
we already use the prefix to get the keys, why would these keys not have that prefix? this condition does not make sense!